$NetBSD$

We actually do want to build the client, even on NetBSD!
--- Makefile.orig	2018-10-14 20:09:09.142621226 -0700
+++ Makefile	2018-10-14 20:09:36.324541351 -0700
@@ -860,7 +860,8 @@
     HAVE_VM_COMPILED=true
   endif
 
-  BUILD_CLIENT = 0
+  CLIENT_LIBS += $(SDL_LIBS)
+  RENDERER_LIBS = $(SDL_LIBS)
 else # ifeq netbsd
 
 #############################################################################
